lib/web_ui/dev/steps/copy_artifacts_step.dart

```diff
@@ -43,7 +43,7 @@ class CopyArtifactsStep implements PipelineStep {
     }
     if (artifactDeps.canvasKitChromium) {
       print('Copying CanvasKit (Chromium)...');
-      await copyCanvasKitFiles('canvaskit_chromium', 'canvaskit/chromium');
+      await copyCanvasKitFiles('canvaskit/chromium', 'canvaskit/chromium');
```

Comment:
why this change?

Reply:
This is a good change. My unit test reorganization has us pulling directly from the location of the original build output, which is changing as we mess around with the toolchain names and stuff. However, now we have copy steps in the build that put everything into the structure we'll see in GCS, so Harry is changing it so it pulls from that location. This is better and easier to maintain.